compiler/rustc_trait_selection/src/traits/wf.rs

```diff
@@ -598,6 +598,9 @@ impl<'tcx> WfPredicates<'tcx> {
                     walker.skip_current_subtree(); // Subtree handled by compute_projection.
                     self.compute_projection(data);
                 }
+                ty::Alias(ty::Inherent, _) => {
+                    // WF if their substs are WF.
```

Comment:
Hm... what about WC on the impl or the associated type itself? Shouldn't we be doing something more like `compute_projection`?

Reply:
I think you are right. I couldn't come up with a soundness bug but I bet that's just because I'm not too familiar with the matter yet. As an aside, interestingly, there are no UI tests for normal projections that go from `pass` to `fail` if I remove the call to `compute_projection` (and `skip_current_subtree` ofc), it "only leads to span changes" in `check-fail` tests.